frontend/package.json

```diff
@@ -77,6 +77,6 @@
     "eslint-plugin-react": "7.30.0",
     "eslint-plugin-react-hooks": "4.5.0",
     "eslint-plugin-simple-import-sort": "7.0.0",
-    "react-error-overlay": "6.0.11"
+    "react-error-overlay": "6.0.9"
```

Comment:
this is annoying - but it is the only way to make sure hot-reload works as expected.

Reply:
alternatively you could just migrate to react-scripts v5 (we had the [same problem in chaos](https://github.com/csesoc/chaos/pull/149), the only [breaking change from 4.x to 5.x](https://github.com/facebook/create-react-app/blob/main/CHANGELOG.md#breaking-changes) that should really matter is the change in eslint parsers)

Reply:
ok surely just ditch CRA and migrate to [vite](https://vitejs.dev) (i am going to peer pressure chaos to do the same)

Reply:
I was shilling for nextjs as well lmao

@gtangelo plz